relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,105 @@
+use std::borrow::Cow;
+use std::sync::Arc;
+use std::time::Duration;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, Interface, Service};
+use reqwest::Method;
+use serde::{Deserialize, Serialize};
+
+use crate::actors::project_cache::ProjectCache;
+use crate::actors::upstream::{RequestPriority, SendQuery, UpstreamQuery, UpstreamRelay};
+
+// No messages are accepted for now.
+#[derive(Debug)]
+pub enum GlobalConfigMessage {}
+
+impl Interface for GlobalConfigMessage {}
+
+/// Service implementing the [`GlobalConfig`] interface.
+///
+/// The service is responsible to fetch the global config appropriately and
+/// forward it to the services that require it.
+#[derive(Debug)]
+pub struct GlobalConfigService {
+    project_cache: Addr<ProjectCache>,
+    upstream: Addr<UpstreamRelay>,
+}
+
+impl Interface for GetGlobalConfig {}
+
+impl GlobalConfigService {
+    pub fn new(project_cache: Addr<ProjectCache>, upstream: Addr<UpstreamRelay>) -> Self {
+        Self {
+            project_cache,
+            upstream,
+        }
+    }
+
+    /// Forwards the given global config to the services that require it.
+    fn update_global_config(&mut self) {
+        let upstream_relay: Addr<UpstreamRelay> = self.upstream.clone();
+        let project_cache = self.project_cache.clone();
+        tokio::spawn(async move {
+            let query = GetGlobalConfig;
+
+            if let Ok(Ok(response)) = upstream_relay.send(SendQuery(query)).await {
+                let global_config = Arc::new(response.global);
+                project_cache.send::<Arc<GlobalConfig>>(global_config);
+            };
+        });
+    }
+}
+
+impl Service for GlobalConfigService {
+    type Interface = GlobalConfigMessage;
```

Comment:
Also this currently does not support any messages, you could just use `()` for the `Interface`

```suggestion
    type Interface = ();
```